components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -691,6 +691,66 @@ public void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey, Properties
         }
     }
 
+    public void revokeTokensAndAuthzCodes(String consumerKey, String[] authorizationCodes, String[] accessTokens)
```

Comment:
Why don't we introduce two simple methods for the two tasks,
1. revoke tokens
2. revoke authorization codes